### PR TITLE
breaking: remove POSTGRES prefix env variables in favor of DATABASE prefix

### PR DESCRIPTION
**Pull request by thetutlage**
FIXES: FRMW-2659

As the title says, we want to use consistent naming for database specific environment variables and decided to stick to the `DATABASE_` prefix and remove all usages of `POSTGRES_` prefix

### CODE REVIEW

File: packages/cli/create-medusa-app/src/utils/prepare-project.ts
Comment (olivermrbl) [inline packages/cli/create-medusa-app/src/utils/prepare-project.ts:76]: 🙏

**Review by olivermrbl — APPROVED** [review]
Nice LGTM, pending @carlos-r-l-rodrigues's approval

**Review by carlos-r-l-rodrigues — APPROVED** [review]

**Review by sradevski — APPROVED** [review]

**Pull request merged**
